### ISSUE TITLE
Different icon in menu-bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#13]
```
All the icons in my menu-bar are black and white, except for the nosleep 
utility one. I would really like it if you could adopt the style used by the 
app 'caffeine'. In this case that would mean that if nosleep is enabled the ZZ 
icon is black. When disabled the ZZ's would be gray with a black outline.

And thank you for this very useful extension!
```

Original issue reported on code.google.com by `bsprec...@gmail.com` on 15 Dec 2011 at 2:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `integral...@gmail.com` on 17 Feb 2012 at 5:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `integral...@gmail.com` on 23 Apr 2012 at 10:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `integral...@gmail.com` on 14 Jun 2012 at 5:31
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 1.3.1
```

Original comment by `integral...@gmail.com` on 15 Aug 2012 at 7:05
- Changed state: **Fixed**
